blog/_posts/2017-09-8-railsgyn-lastblogpost.md

```diff
@@ -0,0 +1,59 @@
+---
+
+
+## All good things come to an end… OH GOD WHY!??!
+We are just shocked! We still cannot believe how fast these three months passed!
+We do not want this summer to finish!
+
+<img src="/img/blog/2017/cry.gif" alt="crying">
+<div align="center" div class="image-credits">That is exactly how we are now!</div>
+
+RGSoC was by far the best place ever that we already worked on. The environment was pure learning and assistance, we could implement a lot of cool features, we could meet a lot of nice people, we could discuss our ideas and we were heard. Certainly, we will keep contributing with the RGSoC team's app, because this program deserves the best management app that could exist and we want to work on it!
+
+<img src="/img/blog/2017/icons.png" alt="rgsoc">
+<div align="center" div class="image-credits">Design by Juliana</div>
+
+## What were our main achievements in this summer?
+
+We started the summer of code working on beginner friendly issues, however, on the very first days, one of our friends from TeamsApp asked for help in a milestone to automate the conference process to offer conference tickets to teams. It was a big change, with a lot of issues and a lot of models builds and refactors. She even though that it wasn’t possible to build for this summer, but we did it! We build almost all the related things to this milestone conclusion and when we saw it on production was proud.
+It was really surrounding construct such a complex issue in a really new project for us. We had to learn a lot in just some days, but worth every second when we saw everything working.
+That is was a challenge do with you, it makes you find out how far you can go.
+After finish the conference milestone, we have decided to start a community milestone and we still are working on it. Even if we did not finish it until september/30 we will finish it! 
+We are so happy to have our fingerprints on this project! That is something priceless.
+
+
+<img src="/img/blog/2017/we.jpg" alt="us">
+<div align="center" div class="image-credits">By: Juliana's mother</div>
+
+## What will we miss?
+* We will miss all the excellent developers from team's app who worked closely with us in this summer: Carsten, Max, Maria, Maud...
+* We will miss Ines :\ our conversations were always the best ones and we don’t want to stop it.
+* We will miss sharing in all the social media that we work on the RGSoC.
+* We will miss pair working in nice coffee places in our city.
+
+## What do we learn?
+Well, this summer was a great learning experience for us, our mentors were fantastic with your code reviews, it makes us grow up a lot. During this summer we can learn more about tests with RSpec, using shared examples, many factories, finite state machine concept, some Ruby methods, code best practices applying DRY,  metaprogramming, service objects, work with Continuous Integration, Git workflow and many ruby facts that we happily shared on our [blog](https://medium.com/@railsgirlsgyn).
```

Comment:
This line: 'our mentors were fantastic with your code reviews, it makes us grow up a lot looks bit odd too.' Can you change/rephrase it as well?